app/helpers/application_helper.rb

```diff
@@ -74,11 +74,24 @@ def records_sorted_by_short_name(records)
     end
   end
 
-  def campaign_select_list
+  def campaign_select_list(form:)
     select_list = [ANY_CAMPAIGN_KEYWORD]
 
-    camps = RecordData.select(:value).where(column_name: "Campaigns/Campaign/ShortName").where.not(value: "").order(:value).distinct
-    select_list.concat(camps.map(&:value))
+    if current_user.daac_curator?
+      daac = current_user.daac
+      state = form == home_path ? :in_daac_review : [:finished, :closed]
+    elsif current_user.mdq_curator?
+      daac = MDQ_PROVIDERS
+      state = form == home_path ? [:open, :in_arc_review, :ready_for_daac_review, :in_daac_review] : [:finished, :closed]
```

Comment:
check to see if in_daac_review should be removed here

Reply:
I don't think a mdc_curator? can see in_daac_review state, but you were going to double check.

Reply:
I've made the change, but I haven't pushed it because I was waiting for other feedback.